.travis.yml

```diff
@@ -17,10 +17,14 @@ before_cache:
 
   - rm -rfv $HOME/.cabal/packages/head.hackage
 
+  # remove files from old compilers
+  - rm -rfv $HOME/.cabal/store/{!(ghc-8.6.5)}
+  - rm -rfv $TRAVIS_BUILD_DIR/dist-newstyle/build/*/{!(ghc-8.6.5)}
+
```

Comment:
We’re taking several minutes just to download and unpack the cache, meaning it’s probably bigger than we might expect. This just clears out the non-8.6.5 contents of those dirs.